src/EPR.Calculator.API/Exporter/CalcResultsExporter.cs

```diff
@@ -381,10 +381,6 @@ private static void PrepareSummaryData(CalcResultSummary resultSummary, StringBu
                 csvContent.Append($"£{CsvSanitiser.SanitiseData(Math.Round(producer.ScotlandTotalWithBadDebtProvision2A, decimalRoundUp))},");
```

Comment:
Do we need £ sign here? Can we use a format instead?

Reply:
@udaydefra - yes we can use formatting here, but it is not related to this PR. There are lot of occurrences of this in this file and changing this will introduce complete regression testing.

I would suggest that we can create an improvement ticket for this work.

Reply:
Could you please create a ticket and put it in the backlog? Thanks